src/main/java/org/sonarqube/scm/cvs/CvsConfiguration.java

```diff
@@ -131,39 +130,39 @@ public CvsConfiguration(Settings settings) {
 
   @CheckForNull
   public String username() {
-    return settings.getString(USER_PROP_KEY);
+    return settings.get(USER_PROP_KEY).orElse(null);
   }
 
   @CheckForNull
   public String password() {
-    return settings.getString(PASSWORD_PROP_KEY);
+    return settings.get(PASSWORD_PROP_KEY).orElse(null);
   }
 
   public boolean compressionDisabled() {
-    return settings.getBoolean(DISABLE_COMPRESSION_PROP_KEY);
+    return settings.getBoolean(DISABLE_COMPRESSION_PROP_KEY).orElseThrow(() -> new IllegalStateException("Missing default value of " + DISABLE_COMPRESSION_PROP_KEY));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal "Missing default value of " 3 times. ([squid:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.scm.cvs%3Asonar-scm-cvs-plugin&pullRequest=6&issues=AWKrn4f9yi-P6iIqlb4i&open=AWKrn4f9yi-P6iIqlb4i)

[ISSUE_KEY]: AWKrn4f9yi-P6iIqlb4i